src/containers/shared/components/Transaction/AMM/AMMDeposit/simple.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { SimpleRow } from '../../SimpleRow'
+import { TransactionSimpleProps } from '../../types'
+import { Account } from '../../../Account'
+import { Amount } from '../../../Amount'
+
+export const Simple = ({ data }: TransactionSimpleProps) => {
+  const { t } = useTranslation()
+  const { amount, amount2, accountID, ePrice, fee } = data.instructions
+  const value =
+    amount?.amount && amount.amount !== fee ? (
+      <Amount value={amount} />
+    ) : undefined
+  const value2 =
+    amount2?.amount && amount2.amount !== fee ? (
+      <Amount value={amount2} />
+    ) : undefined
```

Comment:
What exactly is this code doing?

Reply:
when you make transactions theres a very small fee thats charged that the user doesn't specify. Here I filter those fees out so users arent confused in cases where they only deposit/withdraw one asset. Theyre very small, like 0.0000001xrp.